equinox-shipment/Domain/Container.fs

```diff
@@ -0,0 +1,56 @@
+module Container
+
+type Container = { id: string; shipmentIds: string[]; finalized: bool }
+
+module Events =
+
+    let [<Literal>] CategoryId = "Shipment"
+
+    let (|ForClientId|) (clientId: string) = FsCodec.StreamName.create CategoryId clientId
```

Comment:
see master, we have a new convention - in order to make sure it does not get lost when people add types before `module Events` at file level on L4 we do this as:
```
let [<Literal>] Category = "Shipment"
let streamName id = FsCodec.StreamName.create CategoryId (IdType.toString id)
```

Reply:
Do you have a direct link to a code line on `master`?

Reply:
https://github.com/jet/equinox/commit/ebeb34d4a8cd30872f3d366ff7cc2f70257341f1

Reply:
Updated thanks.